fdbclient/Metacluster.cpp

```diff
@@ -31,6 +33,47 @@ FDB_DEFINE_BOOLEAN_PARAM(RunOnMismatchedCluster);
 FDB_DEFINE_BOOLEAN_PARAM(RestoreDryRun);
 FDB_DEFINE_BOOLEAN_PARAM(ForceJoinNewMetacluster);
 
+namespace TenantAPI {
```

Comment:
As part of the metacluster, we can use the `MetaclusterAPI` namespace rather than the tenant one